### DISCUSSION TITLE
Rusqlite insert data issue

### DISCUSSION THREAD

**Discussion by sakabda** [rusqlite/rusqlite#1287] [Questions]
Hi Rust, I want to insert value in a case insensitive column by using rusqlite, but I can not able to do that, rusqlite not allow me to insert data in case insensitive filed, Please help with that. Thank you

**Comment by gwenn** [comment]
Are you sure ?
https://sqlite.org/lang_naming.html
> SQL identifiers [...] are case-insensitive

Maybe you mean
https://sqlite.org/faq.html#q18
> Case-insensitive matching of Unicode characters does not work.
?

**Reply by sakabda** [reply]
fn create_at_port(
    dailyReptAtPortAeConsAe_10Rhrs : String,
) -> Result<()> {let conn = Connection::open("sqlite.db")?;
    println!("Opened database successfully");
    conn.execute(
        "INSERT INTO atPort (
        dailyReptAtPortAeConsAe_10Rhrs
        ) VALUES (
            ?1
        )",
        params![
            &dailyReptAtPortAeConsAe_10Rhrs
        ],)?;
    println!("User created successfully");
    Ok(())
}
 **Against this column "dailyReptAtPortAeConsAe_10Rhrs", we can not insert value, as rusqlite didn't allow me to do that.
**** Please help us

**Reply by wezm** [reply]
What is the full error you get? (BTW you should wrap code in three backticks to format it properly). E.g.

\`\`\`
`code here`
\`\`\`

**Comment by gwenn** [comment]
```sh
sqlite> CREATE TABLE atPort (dailyReptAtPortAeConsAe_10Rhrs);
sqlite> INSERT INTO atPort (dailyReptAtPortAeConsAe_10Rhrs) VALUES ('test');
sqlite> SELECT * FROM atPort;
test
```
https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=31734ee7af29532f806205f6c522ffd0
```
Opened database successfully
User created successfully
```
